src/spirc.rs

```diff
@@ -246,7 +295,9 @@ impl SpircTask {
 
         if frame.get_ident() == self.ident ||
            (frame.get_recipient().len() > 0 && !frame.get_recipient().contains(&self.ident)) {
-            return;
+            if frame.get_typ() == MessageType::kMessageTypeNotify {
```

Comment:
Thanks! This works:

``` Rust
fn handle_command(&mut self, cmd: SpircCommand) {
        SpircCommand::Pause => {
        if self.device.get_is_active() {
            self.handle_pause();
        }
        self.notify(None);
    }
}
```

Should `self.notify(None);` better be placed inside the `if` branch`?